UPGRADING.md

```diff
@@ -1,6 +1,19 @@
 Upgrading
 =========
 
+From 0.6.3 or older
+-----------------
+
+- Apache's `X-Request-Start` header is now recognized, which if present
+  will trigger the default `wait_timeout` behavior (unless you have changed
+  those settings or disabled it).
+
+  Please review `wait_timeout` behavior in [doc/settings.md](doc/settings.md#wait-timeout).
+
+  TL;DR -> set `wait_timeout` to `0` or `'false'` to disable it and maintain
+  your Apache setup's current behavior.
+
+
```

Comment:
```suggestion
```

I don't think this comment is necessary, as generally this file is for backwards-incompatible changes ("it broke my app") rather than new features.

Reply:
We experienced some unanticipated behavior in our application when we switched from Apache to nginx several months ago, because (unbeknownst to us at the time) rack-timeout had been ignoring the header with Apache but began honoring it with nginx.

So in that sense this could be considered a breaking change.

Reply:
Correct, but that is due to a bug (or even a purposefully missing feature -- not supporting Apache), not because of backwards-incompatible change.

Reply:
OK, makes sense.  I feel like a word of caution to Apache users is warranted; would putting a short version of this in the changelog instead be acceptable?